packages/lib/src/vis/models.ts

```diff
@@ -48,6 +48,9 @@ export interface AxisConfig {
 }
 
 export type VisScaleType = ColorScaleType | [ScaleType.Gamma, number];
+export type ExtractScaleType<V extends VisScaleType> = V extends ColorScaleType
+  ? V
+  : ScaleType.Gamma;
```

Comment:
This is the TS equivalent of extracting the `ScaleType` enum value from a `VisScaleType` parameter (if array: `visScaleType[0]`; else: `visScaleType`).